src/main/java/org/betonquest/betonquest/quest/condition/scoreboard/ScoreboardObjectiveConditionFactory.java

```diff
@@ -11,7 +11,7 @@
 /**
  * Factory to create scoreboard conditions from {@link Instruction}s.
```

Comment:
You need to add the "objective" part here and in the event or just `{@link}` the class there.